framework/PageToolbar/PageToolbarFilters/ToolbarAsyncSingleSelectFilter.tsx

```diff
@@ -18,7 +20,10 @@ export interface IToolbarAsyncSingleSelectFilter extends ToolbarFilterCommon {
   queryErrorText?: PageAsyncQueryErrorTextType;
 
   // useHook for modal here
-  openBrowse?: (onSelect: (value: string) => void, defaultSelection?: string) => void;
+  openBrowse?: openBrowseType;
+
+  // default selected item
+  defaultSelection?: string;
```

Comment:
When would the defaultSelection be specified as part of the filter itself?
It seems like the `openBrowse` should be taking in the current selection, which is loaded from the querystring values for the filter.